corehq/apps/locations/models.py

```diff
@@ -353,7 +353,7 @@ class SQLLocation(MPTTModel):
     metadata = jsonfield.JSONField(default=dict, blank=True)
     created_at = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True)
-    is_archived = models.BooleanField(default=False)
+    is_archived = models.BooleanField(default=False, db_index=True)
```

Comment:
Spun up a little doc about this query: https://docs.google.com/document/d/1wwrxVa3dXJFhMwvm-ehUbyCIUgFI_1XRdyVjlCLFLX0/edit